roles/provision-metrics-apb/tasks/provision-app-metrics.yml

```diff
@@ -0,0 +1,115 @@
+- name: Postgresql PVC
+  k8s_v1_persistent_volume_claim:
+    name: postgresql-data
+    namespace: "{{ namespace }}"
+    labels:
+      app: app-metrics-api
+      service: prometheus
+    access_modes:
+      - ReadWriteMany
+    resources_requests:
+      storage: "10Gi"
+    state: present
+
+- name: create postgres deployment config
+  openshift_v1_deployment_config:
+    name: postgres
+    namespace: '{{ namespace }}'
+    labels:
+      app: postgres
+      service: prometheus
+    replicas: 1
+    selector:
+      app: postgres
+      service: prometheus
+    spec_template_metadata_labels:
+      app: postgres
+      service: prometheus
+    containers:
+    - name: postgresql
+      image: registry.access.redhat.com/rhscl/postgresql-96-rhel7:latest
+      imagePullPolicy: IfNotPresent
+      ports:
+        - container_port: 5432 
+      env:
+      - name: POSTGRESQL_PASSWORD
+        value: postgres
+      - name: POSTGRESQL_USER
+        value: user
+      - name: POSTGRESQL_DATABASE
+        value: aerogear_mobile_metrics
+      volumeMounts:
+        - mountPath: /var/lib/pgsql/data
+          name: postgresql-data
+    volumes:
+      - name: postgresql-data
+        persistentVolumeClaim:
+          claimName: postgresql-data
+
+- name: Create db internal service
+  k8s_v1_service:
+    name: postgres-internal
+    namespace: '{{ namespace }}'
+    labels:
+      app: postgres
+      service: prometheus
+    selector:
+      app: postgres
+      service: prometheus
+    ports:
+      - name: web
```

Comment:
is `web` the right name for this port? Maybe `postgres` as it's using the postgres protocol, not http

Reply:
Good point i'll change that